src/Symfony/Component/Console/Input/InputArgument.php

```diff
@@ -111,6 +113,19 @@ public function getDefault(): string|bool|int|float|array|null
         return $this->default;
     }
 
+    public function setValues(\Closure|iterable|null $values = null)
+    {
+        $this->values = $values;
+    }
+
+    /**
+     * Returns suggestion values for input completion.
+     */
+    public function getValues(): \Closure|iterable|null
```

Comment:
should this return array only? aka resolve the closure in the method?
please don't make this allow iterators, that's unneeded complexity